plugins/orchestrator-common/src/openapidocument.ts

```diff
@@ -0,0 +1,128 @@
+// GENERATED FILE. DO NOT EDIT.
```

Comment:
Since it is auto-generated, I think it'd make sense to move this file to `auto-generated` folder.
Maybe `src/auto-generated/api/definition.ts`
WDYT?